test/test_ft_qualitycheck.m

```diff
@@ -4,6 +4,8 @@
 % WALLTIME 00:30:00
 % DEPENDENCY ft_qualitycheck
 
+ft_hastoolbox('obsolete', 1);
```

Comment:
should this not be `compat/obsolete` just like in test_issue623?

Reply:
yes, that's right. I was too rash